static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTree.tsx

```diff
@@ -668,6 +712,10 @@ export function useVirtualizedTree<T extends TreeLike>(
     return {height, maxHeight: height, overflow: 'hidden'};
   }, [tree.flattened.length, props.rowHeight]);
 
+  // It is important that this is not executed from a map function because
+  // we are assigning the refs to each individual item. If we do that,
+  // we lose access to the refs and cannot call focus or scrollIntoView on them
```

Comment:
Do we need this comment here as well as inside the `useMemo`?

Reply:
Def not, bad merge